cli.js

```diff
@@ -21,13 +21,23 @@ function init(p) {
 	process.exit(p ? 0 : 1);
 }
 
+function initArray(filename, p) {
+	console.log(filename + ': ' + (p ? logSymbols.success + ' Progressive' : logSymbols.error + ' Baseline'));
```

Comment:
The symbol should be before the filename, and we can drop the `Progressive`/`Baseline` as it's implied.